deployment/src/strongmind_deployment/container.py

```diff
@@ -433,19 +438,37 @@ def setup_load_balancer(self, kwargs, project, project_stack, stack):
             ),
         )
 
-        self.load_balancer = awsx.lb.ApplicationLoadBalancer(
-            "loadbalancer",
-            name=project_stack,
-            default_target_group_port=self.container_port,
-            tags=self.tags,
-            access_logs=aws.lb.LoadBalancerAccessLogsArgs(
-                bucket="loadbalancer-logs-221871915463",
```

Comment:
If it was like this before, my comments above are likely not something to worry about.